### ISSUE TITLE
`returnValues` parsed incorrectly

### ISSUE THREAD

**Issue by sebayaki** [ethereum/web3.js#2373]
It's probably related to #2265 
Now it has `returnValues`, but they're incorrect.

#### Steps to reproduce
```
contract.events.Transfer({ fromBlock: 7208005 }, function(error, event) {
   console.log(event);
});
```
#### Expected behavior
The event object should have return value with `from`, `to` and `value` parameters.

For example, this transaction: https://etherscan.io/tx/0x014f39644c60ffa56dd2073ec1783457e93ce8adad8161cf519209cd6b4aadeb

Should have
```
returnValues: { 
     from: '0xc12f436236800293d6fd7a8ede0814940b7da8cf',
     to: '0x326c5e860f602d48b32400438e379be513ccae33',
     value: '6000000000000000000000' 
},
```

#### Actual behavior
Those properties exist, but with incorrect values

```
returnValues: { 
     from: '0xdc278dAAa51ba7f163c4a11628155a4df523b3EF',
     to: '0xc12f436236800293d6fd7a8ede0814940b7da8cf',
     value: '6000000000000000000000'
}
```
it seems `from` value went to `to` parameter and actual `from` value has gone and filled with an incorrect value.

#### Versions
NPM: 6.7.0
Node: 11.7.0
Web3.js: 1.0.0-beta.46
OS: Mac OS 10.14.3

**Comment by mistersandman** [comment]
this is a duplication of #2368

**Comment by sebayaki** [comment]
oh thanks. please merge with #2368 then

**Comment by nivida** [comment]
Closed because of issue #2368.

**Comment by bithalo** [comment]
Request to reopen, newest version of web3 still is giving incorrect return values for structs.
EDIT:
It was due to the difficulty in refreshing page on local testing environment. ABI was included as separate file and didn't update even when I told it to "forget" the page. Needed hard refresh.
